hudi-platform-service/hudi-metaserver/hudi-metaserver-server/pom.xml

```diff
@@ -92,6 +92,34 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>thrift-gen-source-with-script</id>
```

Comment:
Just for my understanding, why do we need a separate profile? Shouldn't the hudi-platform-service profile suffice?

Reply:
In this change, we add a new profile to use maven-thrift-plugin to compile thrift in Azure CI container. In GH CI, we keep the original way since installing thrift would take about 10+ minutes. Therefore, we have two ways for thrift compilation: in Azure CI, we use maven-thrift-plugin; in GH CI, we use original script. To separate them, I create two profiles.

Reply:
do we need to separate them? is it easy to get GH CI to use the plugin as well? if so, lets do it in this same PR

Reply:
IIUC the script must be installing thrift already. We can just replace the build steps with a mvn command.

Reply:
We can check if we can use the same plugin for GH CI. but will do that in a separate PR.